### PR TITLE
Newsletters: add 233 (2023-01-11)

### PR DESCRIPTION
**Pull request by harding**
- [ ] Update releases/RCs, add topic links, add lede @harding 
- [x] Eclair 2455 @Xekyo

### CODE REVIEW

**Review by bitschmidty — COMMENTED** [review]
Looks good, no suggestions

**Review by LarryRuane — COMMENTED** [review]
Nits, otherwise LGTM

File: _posts/en/newsletters/2023-01-11-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    In addition to allowing offline wallets to receive funds for LN, the
+    authors describe how the idea might combine well with [async
+    payments][topic async payments] to allow LSPs to prepare channel
+    rebalance operations in advance for when an offline client comes
+    back online, allowing those rebalance operations to occur without
+    any delay from the user's perspective.  For example, if Carol sends
+    an async LN payment to Alice for an amount larger than the current
+    capacity in Alice's channel, Bob can send a payment to the script
+    `pk(B) && (pk(A) || older(6000)`.  If that payment has a sufficient
+    number of confirmations by the time Alice comes next comes online,
```
Comment (Xekyo) [inline _posts/en/newsletters/2023-01-11-newsletter.md:86]: If Bob sends money to Alice's address in this manner, isn't he just giving her money?

Comment (harding) [reply _posts/en/newsletters/2023-01-11-newsletter.md:86]: This script is different than the script described earlier (the first was `A && (B || timeout)` ; the second is `B && (A || timeout)`.  I'll rephrase to try to make that clearer.

Comment (Xekyo) [reply _posts/en/newsletters/2023-01-11-newsletter.md:86]: Oh yeah, I totally missed that

File: _posts/en/newsletters/2023-01-11-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    In addition to allowing offline wallets to receive funds for LN, the
+    authors describe how the idea might combine well with [async
+    payments][topic async payments] to allow LSPs to prepare channel
+    rebalance operations in advance for when an offline client comes
+    back online, allowing those rebalance operations to occur without
+    any delay from the user's perspective.  For example, if Carol sends
+    an async LN payment to Alice for an amount larger than the current
+    capacity in Alice's channel, Bob can send a payment to the script
+    `pk(B) && (pk(A) || older(6000)`.  This alternative script flips the
```
Comment (jirijakes) [inline _posts/en/newsletters/2023-01-11-newsletter.md:89]:
```suggestion
    `pk(B) && (pk(A) || older(6000))`.  This alternative script flips the
```

File: _posts/en/newsletters/2023-01-11-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    roles for Alice and Bob.  If Bob's payment receives a sufficient
+    number of confirmations by the time Alice comes next comes online,
```
Comment (jirijakes) [inline _posts/en/newsletters/2023-01-11-newsletter.md:91]:
```suggestion
    number of confirmations by the time Alice next comes online,
```

**Comment by harding** [comment]
Updated to address @jirijakes feedback, thanks!

**Comment by bitschmidty** [comment]
Thank you @harding @Xekyo @jirijakes @LarryRuane for contributing this week!

**Pull request merged**